code/modules/hydroponics/grown/nymph.dm

```diff
@@ -11,6 +11,46 @@
 	production = 1
 	yield = 1
 	reagents_add = list("plantmatter" = 0.1)
+	var/twitched = FALSE //Allows ghosts to cause this seed packet to twitch, on a 10 second cooldown.
+
+// If the nymph pods are fully grown, allow ghosts that click on them to spawn as a nymph (decreases yield by 1 and kills the plant when yield reaches 0)
+/obj/item/seeds/nymph/attack_ghost(mob/dead/observer/O)
+	if(!planted_tray)
+		if(!twitched)
+			visible_message("[src] twitches slightly, as if pushed by a gentle breeze.")
+			twitched = TRUE
+			spawn(100)
```

Comment:
We've said this a lot by now, and we shouldn't really have to, anymore. Stop using spawns.